app/models/comment.rb

```diff
@@ -1,4 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
-  belongs_to :post, counter_cache: :comments_counter
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, counter_cache: :comments_counter, foreign_key: :post_id
+  post.increment!(:comments_counter)
```

Comment:
- [ ] Kindly create a method and place the highlighted piece of code inside it. You could name it `update_comments_counter` if you like. This is what the previous reviewer meant 👍

Reply:
@codepantha Hi. The comments are being counted
![image](https://user-images.githubusercontent.com/103516637/203875728-3481d129-1f94-4d5c-8d99-d2d2e43977b2.png)